controllers/helpers/metrics.go

```diff
@@ -0,0 +1,88 @@
+/*
+© 2021 Red Hat, Inc. and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package helpers
```

Comment:
The [guidelines](https://blog.golang.org/package-names) for package naming in Go suggest to avoid general/meaningless names like `util`, `common` et al. I think `helpers` falls into that category as well. I suggest using `metrics` ...